src/styles/content.scss

```diff
@@ -17,6 +19,29 @@ div.os-raise-graybox {
   border-radius: 5px;
 }
 
+div.os-raise-motivation {
+  @extend %os-background;
+  @extend %os-box-formatting;
+}
+
+%os-background {
+  background: $raise-motivation-box;
+  border-radius: 8px;
+  background-image: url("https://k12.openstax.org/contents/raise/resources/c4c17785a634fd81884f3e2f43dc073ad2285ced");
+  background-repeat: no-repeat;
+  background-position: -10px 50%;
+  background-size: 110px;
+}
+
+%os-box-formatting {
+  border: 1px solid $raise-motivation-box;
+  padding: 16px;
+  padding-top: 20px;
+  padding-left: 90px;
+  font-weight: bold;
+  margin-bottom: 10px;
```

Comment:
If we want this margin to be the same as things like `<p>`, we can make it `1rem` instead.